src/Runner/FileFilterIterator.php

```diff
@@ -52,7 +52,12 @@ public function __construct(
 
     public function accept()
     {
+        /** @var \SplFileInfo $file */
         $file = $this->current();
+        if (!($file instanceof \SplFileInfo)) {
```

Comment:
`FileFilterIterator` has to iterate over `SplFileInfo`. if it doesn't, there is sth horribly wrong

Reply:
maybe change to throwing an exception than?

Reply:
how one can pass string here in the first place?